maps/src/maputils.cxx

```diff
@@ -179,6 +198,39 @@ boost::python::tuple GetRaDecMap(G3SkyMapConstPtr m)
 }
 
 
+G3SkyMapPtr GetRaDecMask(G3SkyMapConstPtr m, double ra_left, double ra_right,
+    double dec_bottom, double dec_top)
+{
+	G3SkyMapPtr mask = m->Clone(false);
+	mask->weighted = false;
+	mask->units = G3Timestream::None;
+	mask->pol_type = G3SkyMap::None;
+	mask->SetPolConv(G3SkyMap::ConvNone);
+
+	if (ra_left < 0)
+		ra_left += 360 * G3Units::deg;
+	if (ra_right < 0)
+		ra_right += 360 * G3Units::deg;
```

Comment:
The compiler would generally use mfpr or something to resolve this, but 2*pi radians would be safer than 360 deg. if not. Also, should this be while instead of if, in case ra < -2pi? Or some other truncation?

Reply:
I copied the wrapping logic from the `FlatSkyProjection` class here, which should result in everything being wrapped to [0, 2pi] regardless of sign.